domains/certificates/Query.API/API/MeasurementsSyncer/SlidingWindowService.cs

```diff
@@ -97,26 +102,33 @@ private static bool IsIncludedInMissingInterval(MeteringPointTimeSeriesSlidingWi
         return window.MissingMeasurements.Intervals.Any(missingInterval => missingInterval.Contains(interval));
     }
 
-    public void UpdateSlidingWindow(MeteringPointTimeSeriesSlidingWindow window, List<Measurement> measurements,
-        UnixTimestamp newSynchronizationPoint)
+    public void UpdateSlidingWindow(MeteringPointTimeSeriesSlidingWindow window, List<Measurement> measurements, UnixTimestamp newSynchronizationPoint)
     {
+        var minimumAgeThreshold = CalculateMinimumAgeThreshold();
+
+        if (_options.MinimumAgeThresholdHours > 0 && newSynchronizationPoint > minimumAgeThreshold)
```

Comment:
_options.MinimumAgeThresholdHours > 0 <-- Why are we checking if an options is above 0? Think we could remove this. This should be validating during startup / not start service if improper setup.

Reply:
It is not a validation.

A value of 0 means the following code should not execute, and a value of 0 means the age threshold is disabled.

Reply:
It has been removed. we no longer do this check: https://github.com/Energinet-DataHub/energy-origin/pull/1575/commits/d47746752828caed0a4406bc11f4cb9aa3690974